app/routes/API_Demo/Demo/Demo.js

```diff
@@ -1,37 +1,34 @@
 import React, { useState, useEffect } from "react";
-import {Container} from './../../../components';
+import { Container } from './../../../components';
 
 const Demo = () => {
   const [page, setPage] = useState(0);
   const [stories, setStories] = useState([]);
-  
+
   useEffect(() => {
     fetchData();
   },[]);
 
   async function fetchData() {
```

Comment:
Since we're no longer awaiting on `fetch()`, we no longer need `fetchData()` to be `async`.